Next.js quickstart proposal (#444)
clerkinc/clerk-docs: docs/references/nextjs/read-sessions-data.mdx

---
title: Read session and user data with Clerk and Next.js

> description: Learn how to use Clerk to quickly and easily add secure authentication and user management to your Next.js application. Clerk works seamlessly on both client side and server side components.

Reviewer: ```suggestion
description: Learn how to use Clerk's hooks and helpers to access the active session and user data in your Next.js application.
```

---

> # Read session and user data with Clerk and Next.js

Reviewer: consistent with the other guides.
note: if we update this h1, we need to update the title in the frontmatter to match.

```suggestion
# Read session and user data in your Next.js App with Clerk
```

> # Read session and user data with Clerk and Next.js

Reviewer: consistent with the other guides.
note: if we update this h1, we need to update the title in the frontmatter to match.

```suggestion
# Read session and user data in your Next.js App with Clerk
```
